app/src/main/res/layout/fragment_scheduler.xml

```diff
@@ -8,189 +8,272 @@
+                            android:textSize="@dimen/sp_15"/>
+
+                        <RadioGroup
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            bind:check="@{viewModel.radioButtonId}"
+                            bind:changeListener="@{viewModel.changeListener}">
+
+                            <RadioButton
+                                android:id="@+id/filter_today"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:padding="@dimen/dp_10"
+                                android:text="@string/title_today"/>
+
+                            <RadioButton
+                                android:id="@+id/filter_yesterday"
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:padding="@dimen/dp_10"
+                                android:text="@string/title_yesterday"/>
+
+                            <RadioButton
+                                android:id="@+id/filter_tomorrow"
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:padding="@dimen/dp_10"
+                                android:text="@string/title_tomorrow"/>
+
+                            <RadioButton
+                                android:id="@+id/filter_space_time"
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:padding="@dimen/dp_10"
+                                android:onClick="@{() -> viewModel.onSpaceTimeClick()}"
+                                android:text="@string/title_space_time"/>
+
+                            <RadioButton
+                                android:id="@+id/filter_select_date"
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:padding="@dimen/dp_10"
+                                android:onClick="@{() -> viewModel.onSelectDateClick()}"
+                                android:text="@string/title_select_date"/>
+                        </RadioGroup>
+
+                        <TextView
+                            android:layout_width="match_parent"
+                            android:layout_height="@dimen/dp_1"
+                            android:background="@color/color_grey_400"/>
+
+                        <TextView
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:text="Status"
```

Comment:
WARNING: [HardcodedText] [I18N] Hardcoded string "Status", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.